named-entity-recognition/README.md

```diff
@@ -13,16 +13,34 @@ There are some sources for labeled NER data that can be used to train systems.
 
 You can also create data yourself if it doesn't exist.
 
-TODO: A good procedure to do this should be filled in.
+* Start by collecting unlabelled texts from popular domains like News articles or Wikipedia. If unlabelled texts do not exist, you can translate some sentences. 
+* Start with few sets of entities especially the popular ones like Personal names (PER), organization (ORG), location (LOC) and/or geopolitical entities (GPE). With few sets of entities, one can easily ensure high quality annotation since fine-grained entities and numerous entities can affect the quality of the overall process especially for non-expert annotators. 
+* Familiarize yourself with annotation guides that have been used to develop popular NER datasets like the Message Understanding Conference [(MUC-6) guideline](https://cs.nyu.edu/faculty/grishman/NEtask20.book_1.html) and [LDC guideline](https://www.ldc.upenn.edu/sites/www.ldc.upenn.edu/files/english-entities-guidelines-v6.6.pdf)
+* Look for an easy to use annotation tool like [ELISA](https://aclanthology.org/P18-4001/), [brat](https://brat.nlplab.org/index.html), [Prodigy](https://prodi.gy/), [LightTag](https://www.lighttag.io/), [ioAnnotator](https://ioannotator.com/),  etc. Some provide the tool free for academic use.
+* Create your own annotation guide with examples in the language or domain of interest. Preferrably, with examples from the annotation tool.
+* (Optionally) Create a pre-recorded video on how to use the tool especially if you are working with non-expert annotators. 
+* Recruit odd number of annotators if possible, so that their inter-agreement score can be compared and you can make use of majority vote. Set your required inter-agreement score to be high (e.g at least 0.95) especially for the trial examples. 
```

Comment:
odd number
-> an odd number

majority vote
-> majority voting

"Set your required inter-agreement score to be high" is a bit confusing. It's not clear what the "required inter-agreement score" is (and it's actually "inter-annotator agreement"). Maybe move this mention of inter-annotator agreement to the next bullet.

Reply:
changed inter-agreement score to inter-annotator agreement

the inter-annotator agreement metric is 0.95 Fleiss kappa score. I have updated it